src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +159,186 @@ export default async function() {
 
 Your development server will automatically restart your database whenever this file changes, regenerating your types and seeding your development data from `seed.ts`.
 
+## Connect a libSQL database for production
+
+Astro DB can connect to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server
+
+The way you get these values varies between services, but the next paragraph will show you how can you get started with [Turso](https://turso.tech), a fully managed database platform that uses libSQL.
+
+### Getting started with Turso
+
+You can either use the CLI or the web interface to create and rethrieve the values we need.
```

Comment:
Should we use the first person? I feel like second person is better for guides and first person is better for recipes, but maybe I'm crazy.
@sarah11918 is the best to save us here

```suggestion
You can either use the CLI or the web interface to create and retrieve the values you need.
```

Reply:
I agree with Fryuni, `you` is better here. And this is the recommended form in [Writing and Style Guide](https://contribute.docs.astro.build/guides/writing-style/#writing-style) 😉 

> Do not use we, we’ll, us, let’s etc. (You are not with the reader.)

Reply:
Haha, OF COURSE. We never use "we" unless it's "We (Astro) recommend". I was mostly copying/pasting/moving things around to restructure, so thank you for spotting!

![image](https://github.com/user-attachments/assets/af85025f-bf7d-4842-8c41-efa9a24dd375)